source/uwp/Renderer/lib/AdaptiveFontStylesDefinition.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "AdaptiveCards.Rendering.Uwp.h"
+#include "Enums.h"
+#include "HostConfig.h"
+
+namespace AdaptiveNamespace
+{
+    class AdaptiveFontStylesDefinition
+        : public Microsoft::WRL::RuntimeClass<Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::RuntimeClassType::WinRtClassicComMix>, ABI::AdaptiveNamespace::IAdaptiveFontStylesDefinition>
+    {
+        AdaptiveRuntime(AdaptiveFontStylesDefinition)
```

Comment:
>) [](start = 52, length = 1)

this line should have a semicolon on the end of it. otherwise it confuses clang-format (hence the weird indented public line below)